certified-connectors/Viafirma/apiDefinition.swagger.json

```diff
@@ -0,0 +1,341 @@
+        "description": "Create a sign request and get the permanent link",
+        "operationId": "CreateSignRequest",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json\r",
+            "description": "Content-Type",
+            "x-ms-summary": "Content-Type"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "*/*\r",
+            "description": "Accept",
+            "x-ms-summary": "Accept"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "groupCode": {
+                  "type": "string",
+                  "description": "groupCode",
+                  "title": "",
+                  "x-ms-dynamic-values": {
+                    "operationId": "WhoIAm",
+                    "value-path": "groups-key-item-output",
+                    "value-collection": "groups",
+                    "value-title": "groups-key-item-output"
+                  }
+                },
+                "workflow": {
+                  "type": "object",
+                  "properties": {
+                    "type": {
+                      "type": "string",
+                      "description": "type",
+                      "title": "",
+                      "default": "PRESENTIAL",
+                      "x-ms-visibility": "internal"
+                    }
+                  },
+                  "description": "workflow",
+                  "required": [
+                    "type"
+                  ]
+                },
+                "document": {
+                  "type": "object",
+                  "properties": {
+                    "templateCode": {
+                      "type": "string",
+                      "description": "templateCode",
+                      "title": ""
+                    }
+                  },
+                  "description": "document",
+                  "required": [
+                    "templateCode"
+                  ]
+                }
+              },
+              "default": {
+                "groupCode": "your_group_here",
+                "workflow": {
+                  "type": "WEB"
+                },
+                "document": {
+                  "templateCode": "templateSample"
+                }
+              },
+              "required": [
+                "workflow",
+                "document",
+                "groupCode"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {}
+          },
+          "default": {
```

Comment:
@anfuca Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger.